README.md

```diff
@@ -279,16 +282,35 @@ The content viewset usually doesn't require any additinal routes, so you can lea
 
 ## Remote
 
-A remote knows specifics of the plugin Content to put it into Pulp. Remote defines how to synchronize remote content.
-Pulp Platform provides support for concurrent downloading of remote content. Plugin writer is encouraged to use one of
-them but is not required to.
+Remotes provide metadata about how content should be downloaded into Pulp, such as the URL of the remote source, the download policy, and some authentication settings. The base ``Remote`` class provided by Pulp Platform provides support for concurrent downloading of remote content. Plugin writers are encouraged to use one of them but is not required to.
```

Comment:
Can these be line split to 79 or 100 chars?
Also at the very end: s/is not required/are not required/